src/main/java/mate/academy/EventManager.java

```diff
@@ -1,19 +1,26 @@
 package mate.academy;
 
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
 public class EventManager {
-    public void registerListener(EventListener listener) {
+    private final List<EventListener> listeners = new CopyOnWriteArrayList<>();
```

Comment:
Using CopyOnWriteArrayList ensures thread-safety during iterations when concurrent modifications may occur. However, be aware that write operations like add and remove are expensive because they involve copying the entire underlying array. If frequent modifications are expected, consider other concurrent collection options that better suit the use case.